SB2018/2018-01-31.md

```diff
@@ -0,0 +1,33 @@
+---
+to: All Teams
+subject: Tech day reminder - Saturday 3rd Feb
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on Saturday (3rd feb)!
+
+Tech days are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We may also have a tour of some university facilities as well.
+
+## Location
+The tech day will be held in [Building 25](http://data.southampton.ac.uk/building/25.html) at the University of Southampton's Highfield Campus:
+
+> University Road\
+> Southampton\
+> Hampshire\
+> SO17 1BJ
+
+There's a map attached which shows the recommended parking location.
```

Comment:
Maybe add that we'll have signage from that car-park to Building 25 (only if we actually will of course).

Reply:
We _should_ have signs for this, but I don't think we need to explicitly tell them they exist. They can just appear to help them out